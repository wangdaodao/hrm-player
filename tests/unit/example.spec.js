import { shallowMount } from '@vue/test-utils';
import hrmPlayer from '@/components/index.vue'

describe('hrmPlayer.vue', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallowMount(hrmPlayer, {
      propsData: {
        source: 'rtmp://58.200.131.2:1935/livetv/cctv1',
        type: 'rtmp/flv'
      }
    })
  })
  // 验证props，打印快照
  it('renders', () => {
    expect(wrapper.vm.source).toEqual('rtmp://58.200.131.2:1935/livetv/cctv1')
    expect(wrapper.html()).toMatchSnapshot()
  })
})
