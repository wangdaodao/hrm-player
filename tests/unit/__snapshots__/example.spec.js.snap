// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`hrmPlayer.vue renders 1`] = `
<video data-setup="{}" class="vjs-tech" id="vjs_video_3_html5_api" tabindex="-1">
  <p class="vjs-no-js">
    您的浏览器不支持 video 标签。
  </p>
</video>
`;
